app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} is not"
+                                     f" vaccinated and cannot visit the cafe.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(f"{visitor['name']}'s"
+                                       f" vaccine is"
+                                       f" expired and cannot visit the cafe.")
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(f"{visitor['name']} is"
+                                      f" not wearing a"
+                                      f" mask and cannot visit the cafe.")
+
+        return f"Welcome to {self.name}"
```

Comment:
The return statement should be consistent with the task description. It should return a string formatted as `"Welcome to {self.name}"` only if all conditions are met. The current implementation is correct, but ensure that it aligns with the task requirements.